cli/command/task_command.go

```diff
@@ -160,8 +161,28 @@ func (t *TaskCommand) Delete(c *cli.Context) error {
 }
 
 func (t *TaskCommand) Get(c *cli.Context) error {
+	taskSummaries, err := t.Client.ListTasks()
+	if err != nil {
+		return err
+	}
+
+	taskExists := func(id string) bool {
+		for _, t := range taskSummaries {
+			if t.TaskID == id {
+				return true
+			}
+		}
+
+		return false
+	}
+
 	tasks := []*models.Task{}
 	getTaskf := func(id string) error {
+		if !taskExists(id) {
+			log.Debugln("No corresponding Task found for TaskID:", id)
```

Comment:
yes, explicitness in terms of which print 'verb' (their term) we use: `%s` vs `%v` vs `%#v` etc. etc.